### PR TITLE
Problem: Missing raw transaction projection

### PR DESCRIPTION
**Pull request by davcrypto**
Resolved: #723

### CODE REVIEW

**Review by vincentysc — APPROVED** [review]

File: projection/raw_transaction/raw_transaction.go [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+						Type:           transactionCreatedEventSigner.MaybeKeyInfo.Type,
+						IsMultiSig:     transactionCreatedEventSigner.MaybeKeyInfo.IsMultiSig,
+						Pubkeys:        transactionCreatedEventSigner.MaybeKeyInfo.Pubkeys,
+						MaybeThreshold: transactionCreatedEventSigner.MaybeKeyInfo.MaybeThreshold,
+					}
+				}
+				signers = append(signers, transactionRowSigner)
+			}
+			tx.Signers = signers
+			txs = append(txs, tx)
```
Comment (calvinaco) [inline projection/raw_transaction/raw_transaction.go:128]: I remember we have a conversation about signer info maybe missing in some cases. Can you shared again the conclusion? Is it that the signer always present?

File: projection/raw_transaction/raw_transaction_integration_test.go [hunk 2]
```diff
@@ -0,0 +1,135 @@
+package raw_transaction_test
```
Comment (calvinaco) [inline projection/raw_transaction/raw_transaction_integration_test.go:1]: do we need this file? Seems it is copying from the block projection.

File: projection/raw_transaction/raw_transaction_integration_test.go [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+	model_usecase "github.com/crypto-com/chain-indexing/usecase/model"
+)
+
+var _ = Describe("RawTransaction", func() {
+	It("should implement projection", func() {
+		fakeLogger := NewFakeLogger()
+		fakeRdbConn := rdb_test.NewFakeRDbConn()
+		var _ entity_projection.Projection = block.NewBlock(fakeLogger, fakeRdbConn, nil)
+	})
+
```
Comment (calvinaco) [inline projection/raw_transaction/raw_transaction_integration_test.go:29]: This is some old naive tests I have back in the days. A better way could be just have a line in the source code file:
```
var _ entity_projection.Projection = &RawTransaction{}
```

File: projection/raw_transaction/raw_transaction.go (hunk 1 above)
Comment (davcrypto) [reply projection/raw_transaction/raw_transaction.go:128]: empty signer would be exist here since we do not process any data inside the messages

**Pull request merged**